packages/algob/test/internal/core/config/config-loading.ts

```diff
@@ -169,4 +190,79 @@ describe("config loading", function () {
       ).catch((err) => console.log(err)); ;
     });
   });
+
+  describe("load kmd accounts", function () {
+    useFixtureProject("config-project");
+    useEnvironment();
+
+    const fakeKmd: Kmd = {} as Kmd; // eslint-disable-line @typescript-eslint/consistent-type-assertions
+    const kmdOp = new KMDOperatorMock(fakeKmd);
+    let net: NetworkConfig;
+
+    beforeEach(function () {
+      net = this.env.config.networks.kmdNet;
+    });
+
+    afterEach(function () {
+      kmdOp.resetAccounts();
+    });
+
+    it("should ignore if kmd config is not defined", async function () {
+      net.kmdCfg = undefined;
+      const result = await _loadKMDAccounts(net, kmdOp);
+      assert.isUndefined(result);
+    });
+
+    it("should not connect to kmd if config is invalid", async function () {
+      const invalidKmdCfg = net.kmdCfg as KmdCfg;
+      invalidKmdCfg.port = 123; // invalid port
+
+      expectBuilderErrorAsync(
+        () => _loadKMDAccounts(net, new KMDOperator(createKmdClient(invalidKmdCfg))),
+        ERRORS.KMD.CONNECTION
```

Comment:
this is not implemented as per task, i think we need to have mock client.
also if we are using createKmdClient. this test will always fail 
therefore we cannot test it correctly

Reply:
+ accounts are loaded only when kmdCfg for given network is specified and only if it can connect to the KMD -task

can you please add loading of accounts from a particular network ?

Reply:
(reply to previous comment) I think i am successfully testing the kmd connection here. We are creating an invalid kmdClient (which we can. we discussed this, will reference the discussion here). And passing this client to an instance of parent kmdOp (not base - which is the mock). `Algosdk` allows us to make clients (`algodv2`, `kmd`), but it won't allow us to execute chain operations (since i am adding an invalid port, it should and does give `ECONNREFUSED`, which we are asserting here). Please let me know if we still need to change it.
Reference -  https://github.com/scale-it/algorand-builder/pull/108#issuecomment-717117508

Reply:
yes, understood, we have `mock operator` but not `mock client`.
i was looking at `AlgoOperatorDryRunImpl` for reference , this has different way of testing.
let's keep it same.